shadows/framework/src/main/java/org/robolectric/shadows/ShadowAudioTrack.java

```diff
@@ -384,6 +386,21 @@ public void setPlaybackParams(@Nonnull PlaybackParams params) {
     playbackParams = checkNotNull(params, "Illegal null params");
   }
 
+  /**
+   * Sets the estimated latency of this {@link AudioTrack} that will be returned by {@code
+   * AudioTrack.getLatency()}, in milliseconds.
+   */
+  @RequiresApi(LOLLIPOP)
```

Comment:
The min sdk is already 21, so all these `LOLLIPOP` checks are not necessary, right?

Reply:
Updated in #10157